elk/elasticsearch/config/elasticsearch.yml

```
@@ -0,0 +1,9 @@
+---
+## Default Elasticsearch configuration from Elasticsearch base image.
+## https://github.com/elastic/elasticsearch/blob/main/distribution/docker/src/docker/config/elasticsearch.yml
+#
+cluster.name: docker-cluster
+network.host: 0.0.0.0
+
+xpack.license.self_generated.type: trial
+xpack.security.enabled: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)